### PR TITLE
gau: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/gau.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# gau
+
+> Fetches known URLs from AlienVault's Open Threat Exchange, the Wayback Machine, and Common Crawl of domain(s).
```
Comment (vitorhcl) [inline pages/common/gau.md:3]:
```suggestion
> Get All URLs: fetch known URLs from AlienVault's Open Threat Exchange, the Wayback Machine, and Common Crawl for any domains.
```
- Maybe we can put the acronym before the description and remove the line below it?
- We always use the imperative mood over the indicative mood
- I think "for" is better than "of" in this context
- We don't generally use "(s)" in the pages, we don't need to be that specific for this aspect of the commands. I've added an any to don't make it weird.

**Review by vitorhcl — APPROVED** [review]
Thank you! LGTM after my suggestions

**Review by einverne — APPROVED** [review]
LGTM.

File: pages/common/gau.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# gau
+
+> Get All URLs: fetch known URLs from AlienVault's Open Threat Exchange, the Wayback Machine, and Common Crawl for any domains.
+> GAU stands for Get All URLs.
```
Comment (fazlearefin) [inline pages/common/gau.md:4]:
```suggestion
```

This line looks redundant.

Comment (vitorhcl) [reply pages/common/gau.md:4]: Yeah, we can remove it now.

**Pull request merged**